src/components/settings.svelte

```diff
@@ -23,5 +24,7 @@
 
 	<div class="divider" />
 
+	<SettingsWakelock />
```

Comment:
Maybe check here if wakelock is available and if not, the user cant even see the option?